docs/spark/spark-queries.md

```diff
@@ -332,4 +332,15 @@ Metadata tables can be loaded in Spark 2.4 or Spark 3 using the DataFrameReader
 spark.read.format("iceberg").load("db.table.files").show(truncate = false)
 // Hadoop path table
 spark.read.format("iceberg").load("hdfs://nn:8020/path/to/table#files").show(truncate = false)
-```
\ No newline at end of file
+```
+
+You can also inspect Iceberg metadata tables with the time travel feature:
+
+```scala
+// get table's all data files and each data file's metadata at snapshot-id 7277403863961056344
+spark.read
+        .format("iceberg")
+        .option("snapshot-id", 7277403863961056344L)
+        .load("db.table.files")
+        .show()
+```
```

Comment:
[question] There is a dedicated `TimeTravel` section above, should we move this there ? Your thoughts ?

Reply:
Yes adding to that section also works. But my concern was that, the `TimeTravel` section is before metadata tables section. By then, readers may not understand what the table name `db.table.files` is for.

Reply:
+1 for show it here.